enforcers/kube_enforcer/kubernetes_and_openshift/manifests/kube_enforcer/001_kube_enforcer_config.yaml

```diff
@@ -122,11 +122,29 @@ rules:
   - apiGroups: ["aquasecurity.github.io"]
     resources: ["configauditreports", "clusterconfigauditreports"]
     verbs: ["get", "list", "watch"]
-    #### Can be removed if your platform isn't Openshift
```

Comment:
I think we should not remove this, this will not create problem for normal env.
let this be as it is.